tests/trial_tests/test_trial.py

```diff
@@ -79,7 +79,8 @@ def test_check_distribution_suggest_uniform(storage_mode: str) -> None:
             trial.suggest_uniform("x", 10, 30)
 
         # we expect exactly one warning (not counting ones caused by deprecation)
-        assert len([r for r in record if r.category is not FutureWarning]) == 1
+        warnings = [r for r in record if r.category is not FutureWarning]
```

Comment:
`test_trial.py` imports `warnings` package at L12. How about renaming the variable like below?

```suggestion
        warning_msgs = [r for r in record if r.category is not FutureWarning]
```

Reply:
How about `warning_records`? It is not the message itself.

Reply:
Sounds better to me 👍